packages/core/tests/integration/components/visualization-config/line-chart-test.js

```diff
@@ -67,4 +87,27 @@ module('Integration | Component | visualization config/line chart', function(hoo
 
     await run(() => click('.mock'));
   });
+
+  test('displayStackOption', async function(assert) {
+    assert.expect(2);
+
+    await render(Template);
+
+    assert.ok(
```

Comment:
Can we use assert.dom like we do here: https://github.com/yahoo/navi/blob/f99912bd4facb9f7183ca570eec248589705cb59/packages/dashboards/tests/integration/components/dashboard-filters-expanded-test.js#L60